include/image_undistort/stereo_undistort.h

```diff
@@ -28,6 +28,14 @@ constexpr int kQueueSize = 10;
 // true to load input cam_info from ros parameters, false to get it from a
 // cam_info topic
 constexpr bool kDefaultInputCameraInfoFromROSParams = true;
+// source of output camera information, the options are as follows-
+//  "auto_generated": (default), automatically generated based on the input, the
+//    focal length is the average of fx and fy of the input, the center point is
+//    in the center of the image, R=I and only x-translation is preserved.
+//    Resolution is set to the largest area that contains no empty pixels. The
+//    size of the output can also be modified with the "scale" parameter
+//  "match_input": intrinsic values are taken from the input camera parameters
```

Comment:
Is taking the exact input intrinsics desirable for a stereo pair? At the very least I think you will have to ensure that both cameras are rectified using the same focal length or you will probably get pretty poor stereo depth. One possible solution could be to arbitrarily pick one camera and use its intrinsics to set the output for both cameras.

Reply:
this is a copy-paste of the documentation of the parameter from the other header. the step which obtains an equal fov from both cameras and so on is still performed. my change mostly sets the resolution the same as the input. otherwise, the current code produces an image which can become much larger (particulary, wider) than the input, due to how the undistortion works. this leads to wasted computation if this bigger image is processed. this is specially important for embedded image processing. moreover, this produces an output similar to what you obtain with opencv and is what most users expect (see #50).

in any case, I've only added an option and the default is the same, so I doesn't hurt.

Reply:
I just went back and checked, your using the function setOutputFromInput, which sets all intrinsic and extrinsic camera parameters to match the input, not just the image size. 

The best approach is probably the creation of a new function that just fixes the focal length, updates image size and moves the principle to the center.